src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -170,30 +142,15 @@ namespace AppInstaller::CLI::Workflow
         const auto& installer = context.Get<Execution::Data::Installer>();
         auto info = context.Reporter.Info();
 
-        info << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstaller << std::endl;
+        ShowSingleLineField(info, Resource::String::ShowLabelInstaller, " "_liv);
         if (installer)
         {
-            info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerType << ' ' << Manifest::InstallerTypeToString(installer->InstallerType) << std::endl;
-            if (!installer->Locale.empty())
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerLocale << ' ' << installer->Locale << std::endl;
-            }
-            if (!installer->Url.empty())
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerUrl << ' ' << installer->Url << std::endl;
-            }
-            if (!installer->Sha256.empty())
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerSha256 << ' ' << Utility::SHA256::ConvertToString(installer->Sha256) << std::endl;
-            }
-            if (!installer->ProductId.empty())
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerProductId << ' ' << installer->ProductId << std::endl;
-            }
-            if (!installer->ReleaseDate.empty())
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerReleaseDate << ' ' << installer->ReleaseDate << std::endl;
-            }
+            ShowSingleLineField(info, Resource::String::ShowLabelInstallerType, Manifest::InstallerTypeToString(installer->InstallerType), true);
+            ShowSingleLineField(info, Resource::String::ShowLabelInstallerLocale, installer->Locale, true);
+            ShowSingleLineField(info, Resource::String::ShowLabelInstallerUrl, installer->Url, true);
+            ShowSingleLineField(info, Resource::String::ShowLabelInstallerSha256, (installer->InstallerType == Manifest::InstallerTypeEnum::MSStore) ? "" : Utility::SHA256::ConvertToString(installer->Sha256), true);

```

Comment:
The check should be based on the value at hand, not inferred from the installer type.